test/checkTest.spec.ts

```diff
@@ -0,0 +1,23 @@
+import Vorpal from "vorpal";
+import * as checkTest from "../src/commands/checkTest";
+import { CheckUtils } from "../src/classes/check-utils";
+import { CheckSet } from "../src/interfaces/types";
+
+it('exports itself', () => {
+  const vorpal = new Vorpal();
+  vorpal.use(checkTest.test).execSync= jest.fn().mockImplementation();
+  vorpal.execSync = jest.fn();
+  vorpal.use(checkTest.test).execSync("test");
+  vorpal.execSync("exit");
+
+  expect((vorpal.use(checkTest.test).execSync)).toReturn();
+  expect((vorpal.execSync)).toReturn();
+});
+
+it('checks for prequisites', async () => {
+  const test : CheckSet[] = [{checkable : "git", details : {}}]
```

Comment:
What does this actually test?

Reply:
CheckTest is currently a stub function that will run checks for projects in the future. As CheckUtils class will have more functionality in the future, the .spec will have more functionality in another issue as well.